### PR TITLE
Merge branch 'main' of github.com:kredo-school/13th_yippemee into tables

### PR DESCRIPTION
**Pull request by Ringroll001**
Connect our website with google login

### CODE REVIEW

**Review by keanbillyfernandez — APPROVED** [review]
Can you explain what does the Jetstream do? Thanks!!

**Comment by Ringroll001** [comment]
> Can you explain what does the Jetstream do? Thanks!!

I'm using Jetstream to connect our website with google and Facebook login.

File: .env.example [hunk 1]
```diff
@@ -19,7 +19,7 @@ BROADCAST_DRIVER=log
 CACHE_DRIVER=file
 FILESYSTEM_DRIVER=local
 QUEUE_CONNECTION=sync
-SESSION_DRIVER=file
```
Comment (Ninit29) [inline .env.example:22]: no need to include on the PR, please remove

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 2]
```diff
@@ -14,19 +14,15 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-
             $table->id();
             $table->string('name');
             $table->string('email')->unique();
-            $table->string('username')->nullable( );
-            $table->string('location')->nullable( );
```
Comment (Ninit29) [inline database/migrations/2014_10_12_000000_create_users_table.php:22]: do you intend to remove the username, location, avatar and introduction? Is this not being used by other team member?

File: .env.example (hunk 1 above)
Comment (Ringroll001) [reply .env.example:22]: How can I fix it?

**Review by keanbillyfernandez — APPROVED** [review]

**Pull request merged**